internal/sirius/person.go

```diff
@@ -62,13 +62,3 @@ func (c *Client) Person(ctx Context, id int) (Person, error) {
 
 	return v, err
 }
-
-func FilterInactiveAttorneys(actors []Person) []Person {
-	var activeActors []Person
-	for _, actor := range actors {
-		if actor.SystemStatus {
-			activeActors = append(activeActors, actor)
-		}
-	}
-	return activeActors
-}
```

Comment:
I decided to move this to case.go as we're always going to want to remove all inactive attorneys on the case, and therefore we can just run the filter on the whole case rather than passing in the which persons to filter as a parameter